blog-server-services/src/traits/post_service.rs

```diff
@@ -20,7 +27,7 @@ pub struct Post {
     pub author_slug: String,
     pub author_first_name: Option<String>,
     pub author_last_name: Option<String>,
-    pub tags: String,
+    pub tags: Option<Vec<Tag>>,
```

Comment:
[question] here really need option?
Can be as empty vector?

Reply:
Yes i used _#[serde(default)]_ for dtos tag field so it will initialize it with an empty vector (cz we get tags in separate query)
without it we'll get
![image](https://github.com/tikitko/blog-server/assets/52403678/29ddb17f-6b18-4a48-b46a-269053011096)